plutus-tx/src/PlutusTx/ErrorCodes.hs

```diff
@@ -55,6 +55,8 @@ plutusPreludeErrorCodes = Map.fromDistinctAscList
   , ("PT18", "PlutusTx.Enum.Ordering.toEnum: bad argument")
   , ("PT19", "PlutusTx.List.last: empty list")
   , ("PT20", "PlutusTx.Ratio.recip: reciprocal of zero")
+  , ("PT21", "PlutusTx.List.indexBuiltinList: negative index")
+  , ("PT22", "PlutusTx.List.indexBuiltinList: index too large")
```

Comment:
FYI this list is also [duplicated in docs](https://github.com/IntersectMBO/plutus/blob/27fd26250e1d45583fe95bf06b351f20357f85c1/doc/read-the-docs-site/troubleshooting.rst#L98), I suggest you update the docs too (as per instruction on line 21)